src/visualization/NetworkVolumePlot.vue

```diff
@@ -29,15 +29,20 @@ const store: any = {
   loadingText: 'MATSim Volume Plot',
```

Comment:
Since I didn't have suitable input files I didn't fully test my changes in this component @billyc please make sure it still works